### PR TITLE
Describe by startups need to do more

### PR DESCRIPTION
**Pull request by jayp**

### CODE REVIEW

**Comment by jayp** [comment]
I actually cutting out a lot because it didn't fit. I think we'll have to one more sweep just to make sure it all flows.

File: src/content/200-process.md [hunk 1]
```diff
@@ -5,7 +5,19 @@ short: The Process
...
+
+## Cultivating user feedback
+
+It's not easy getting users to be forthcoming about their feelings and motivations. The ground reality is that there is a dearth of feedback -- and most teams will only get feedback from a self-selected set of users. However, a higher volume of user feedback can be generated by nurturing it consistently. It is important to do the basics right: responding quickly to user feedback (showing equal effort in response as they did in their request), fix broad-impact bugs quickly, and for things that can't be done immediately, it is critical to [keep customers in the loop](/process/engage-with-customers-about-their-feedback) when you actually make progress on a solution. When you earn users' trust by responding diligently to their feedback, tracking it, and centering your development process around it, they'll provide you with more feedback in a virtuous cycle that keeps on giving.
```
Comment (nilkanthjp) [inline src/content/200-process.md:12]: It is important to do the basics right -> It is important to get the basics right?

Comment (nilkanthjp) [reply src/content/200-process.md:12]: fix broad-impact bugs -> fix broadly-impacting bugs

File: src/content/200-process.md [hunk 2]
```diff
@@ -5,7 +5,19 @@ short: The Process
...
+
+Talking to users and diligently tracking their feedback also has addtional benefits:
+
+- helps you qualify the user's pain better - is it a must-have or a nice-to-have. Approaches taken by larger companies with public voting boards are not appropriate tool for startups to listen to their users. These boards let anyone vote for a problem or idea without qualification of their pain — users easily "+1" ideas they think would be "cool". You'll get a lot of feedback, but lot of it will be unqualified.
```
Comment (nilkanthjp) [inline src/content/200-process.md:16]: better - is it a must-have or a nice-to-have. -> better—is it a must-have or a nice-to-have?

Comment (nilkanthjp) [reply src/content/200-process.md:16]: voting boards are not appropriate tool -> voting boards are not an appropriate tool

Comment (nilkanthjp) [reply src/content/200-process.md:16]: qualification of their pain — users easily -> qualification of their pain. Users easily

Comment (nilkanthjp) [reply src/content/200-process.md:16]: - helps you qualify -> - Helps you qualify

File: src/content/200-process.md [hunk 3]
```diff
@@ -5,7 +5,19 @@ short: The Process
...
+
+- if and when your company has to change it's strategy slightly, i.e., change the definition of qualified feedback, you can still get value out of feedback that was recorded previously. In fact, due to this, your team can even run small-scale experiments on appeasing different personas and seeing what impact it makes on growth or engagement.
```
Comment (nilkanthjp) [inline src/content/200-process.md:18]: - if and when your -> - If and when your

**Pull request merged**